aspnetcore/blazor/components.md

```diff
@@ -521,6 +521,44 @@ The [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/blazor/co
 
 The base class should derive from `ComponentBase`.
 
+## Import components
+
+The namespace of a component authored with Razor is based on:
+
+* The project's `RootNamespace`.
+* The path from the project root to the component. For example, `ComponentsSample/Pages/Index.razor` is in the namespace `ComponentsSample.Pages`. All components in the same folder, `Pages`, and the parent folder, `ComponentsSample`, are in scope.
```

Comment:
I think the third sentence here can be modified to something like 
"Components follow c# name binding rules. So in the case of `Index.razor`, all components in the same folder, `Pages`, and the parent folder, `ComponentsSample`, are in scope."